keyboards/deltasplit75/v2/v2.h

```diff
@@ -0,0 +1,37 @@
+#ifndef V2_H
+#define V2_H
+
+#include "../deltasplit75.h"
```

Comment:
The keyboard folder should be in the path, so I think this can just be `deltasplit75.h` - does that not work?